cmd/dagger/engine.go

```diff
@@ -195,13 +196,7 @@ func inlineTUI(
 		return err
 	}
 	defer sess.Close()
-	before := time.Now()
 	err = fn(ctx, sess)
-	program.Send(progrock.StatusInfoMsg{
-		Name:  "Duration",
-		Value: time.Since(before).Truncate(time.Millisecond).String(),
-		Order: 3,
-	})
```

Comment:
This isn't needed now that Progrock shows a timer by default. Note the redundant timing info:

![image](https://github.com/dagger/dagger/assets/1880/b969da6b-0dc2-40c7-80f0-29721fc9a462)